packages/experimental/Avatar/src/JSAvatar.styling.ts

```diff
@@ -80,19 +75,20 @@ export const stylingSettings: UseStylingOptions<JSAvatarProps, AvatarSlotProps,
           !avatarColor || AvatarColors.includes(avatarColor as AvatarNamedColor) || ColorSchemes.includes(avatarColor as AvatarColorSchemes)
             ? backgroundColor
             : avatarColor;
+        const ringConfig = getRingConfig(tokens);
 
         return {
           style: {
             ...borderStyles.from(tokens, theme),
             width: size,
             height: size,
-            flexGrow: 1,
-            alignSelf: 'stretch',
             justifyContent: 'center',
             alignItems: 'center',
             backgroundColor: _avatarColor,
             borderWidth: tokens.borderWidth,
             borderColor: tokens.borderColor,
+            marginTop: ringConfig.ringThickness,
+            marginLeft: ringConfig.ringThickness,
```

Comment:
getRIngConfig depends on ringThickness, so it should be added to the caching array

Reply:
Same comment for image below